workspaces/config/test/definitions/definitions.js

```diff
@@ -814,11 +814,10 @@ t.test('save-exact', t => {
     'save-exact': true,
     'save-prefix': '~1.2.3',
   }
-  let flat = {}
+  const flat = {}
   mockDefs()['save-exact']
     .flatten('save-exact', { ...obj, 'save-exact': true }, flat)
-  t.strictSame(flat, { savePrefix: '', save: true })
-  flat = {}
```

Comment:
flatten modifies the object passed in, which is why it's reset like this: so we can be sure that the assertion isn't passing because of previous state.  In this case it's obviously impossible, but the pattern is kept to prevent future problems if we iterate on these tests at all.  Perhaps two distinct variables are more appropriate.

Reply:
I see, it looks like that whole test file could use a good audit on that front, not gonna block this because of it.